central/pruning/pruning.go

```diff
@@ -806,6 +809,27 @@ func (g *garbageCollectorImpl) removeRisks(riskType storage.RiskSubjectType, ids
 	}
 }
 
+func (g *garbageCollectorImpl) removeOrphanedComponents() {
+	if features.PostgresDatastore.Enabled() {
+		pruneStmt := `DELETE FROM active_components child WHERE NOT EXISTS 
```

Comment:
It would be better to move these to central/activecomponent/datastore with other Postgres operations.

Reply:
Yeah I suppose that is the correct thing to do, just more cumbersome to get there.